source/runner/Executor.ts

```diff
@@ -1,23 +1,16 @@
-// @flow
-
 import { contextForDanger } from "../runner/Dangerfile"
 import { DangerDSL } from "../dsl/DangerDSL"
-import type { CISource } from "../ci_source/ci_source"
+import { CISource } from "../ci_source/ci_source"
 import { Platform } from "../platforms/platform"
-import type { DangerResults } from "../dsl/DangerResults"
-import githubResultsTemplate from "./templates/github-issue-template"
+import { DangerResults } from "../dsl/DangerResults"
+import { template as githubResultsTemplate } from "./templates/github-issue-template"
 import { createDangerfileRuntimeEnvironment, runDangerfileEnvironment } from "./DangerfileRunner"
-import type { DangerfileRuntimeEnv } from "./types"
+import { DangerfileRuntimeEnv } from "./types"
 
 // This is still badly named, maybe it really should just be runner?
 
-export default class Executor {
```

Comment:
is `export default` something that is done less in TS? I've only worked in vscode's source, and wrote my own projects from scratch - so I don't have much ambient cultural knowledge